src/@types/Asset.ts

```diff
@@ -90,6 +90,25 @@ export interface AssetDatatoken {
   serviceId: string
 }
 
+export interface AssetPrice {
+  /**
+   * The price of the asset expressed as a number. If 0 then the price is FREE.
+   * @type {number}
+   */
+  value: number
+
+  /**
+   * The symbol that the price of the asset is expressed in.
+   * @type {string}
+   */
+  tokenSymbol?: string
```

Comment:
why is this optional? And also `tokenAddress` and the whole `price` object? Can I expect it to be present on all responses, or not? Marking it optional means that some requests will have it, and some don't

Reply:
Currently Aquarius doesn't send `tokenSymbol` or `tokenAddress` when the asset is free. It just sends ` "price": { "value": 0.0}` so they would need to remain optional unless we change the response that Aquarius sends. I'll make the whole price object required though as it should always be there